dynawo/sources/API/PAR/xsd/parameters.xsd

```diff
@@ -19,14 +19,15 @@
 
   <xs:element name="parametersSet">
     <xs:complexType>
-      <xs:sequence>
+      <xs:choice minOccurs="0" maxOccurs="unbounded">
```

Comment:
a choice means that only one of the elements is given, you have to use sequence here if I'm not mistaken